apps/hash-graph/Cargo.toml

```diff
@@ -48,6 +48,9 @@ tokio = { workspace = true }
 tokio-postgres = { workspace = true }
 tokio-util = { workspace = true, features = ["codec"] }
 tracing = { workspace = true }
+harpc-server.workspace = true
+multiaddr.workspace = true
```

Comment:
```suggestion
harpc-server = { workspace = true }
multiaddr = { workspace = true }
```

Reply:
see: [03a1ceb](https://github.com/hashintel/hash/pull/5610/commits/03a1cebc399918262c2176fe80e7af9ed8d89e59) (i forgot to push that one last night^^)